src/tests/components/Navbar.test.js

```diff
@@ -13,14 +13,14 @@ describe('Navbar', () => {
       expect(wrapper.find('Navbar').length).toEqual(1)
     })
 
-    it('renders 6 MenuItem elements', () => {
+    it('renders 7 MenuItem elements', () => {
```

Comment:
I am not 100% sure that this test make sense anymore. My vote is to remove the this test and the one bellow that check the links rendered

Reply:
Sorry, I didn't mean to mark this as resolved... I wanted to make a comment... I agree with you @joaopapereira... it's annoying to need to update this every time we add another link. 
Also, we will be testing that the link exists by clicking on it in cypress tests